ui/raidboss/popup-text.ts

```diff
@@ -69,14 +64,16 @@ export type ProcessedTrigger = LooseTrigger & {
   output?: Output;
 };
 
-type ProcessedTimelineTrigger = ProcessedTrigger & {
-  isTimelineTrigger?: true;
-};
+// a loaded and localized trigger
+export type LocalizedTrigger =
+  & LooseTrigger
+  & { filename: string; output?: Output }
+  & ({ localRegex: RegExp } | { localNetRegex: RegExp } | { disabled: true });
 
-type ProcessedTriggerSet = LooseTriggerSet & {
-  filename?: string;
-  timelineTriggers?: ProcessedTimelineTrigger[];
-  triggers?: ProcessedTrigger[];
+export type LoadedTriggerSet = LooseTriggerSet & {
+  filename: string;
+  timelineTriggers?: LooseTimelineTrigger[];
+  triggers?: LooseTrigger[];
```

Comment:
```suggestion
```

Aren't these already defined in `LooseTriggerSet`? I'm not sure I understand why they need to be re-listed here.

Reply:
`LooseTriggerSet` now is `triggers?: (LooseTrigger & LegacyTrigger)[];`

and
```ts
export type LegacyTrigger = {
  regexEn?: RegExp;
  regexDe?: RegExp;
  regexFr?: RegExp;
  regexCn?: RegExp;
  regexJa?: RegExp;
  regexKo?: RegExp;

  // @deprecated
  netRegexEn?: RegExp;
  netRegexFr?: RegExp;
  netRegexDe?: RegExp;
  netRegexCn?: RegExp;
  netRegexJa?: RegExp;
  netRegexKo?: RegExp;
};
```